src/pages/iou/MoneyRequestDescriptionPage.js

```diff
@@ -45,11 +45,12 @@ const propTypes = {
     }).isRequired,
 
     /** The current tab we have navigated to in the request modal. String that corresponds to the request type. */
-    selectedTab: PropTypes.oneOf([CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]).isRequired,
+    selectedTab: PropTypes.oneOf(['', CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]),
 };
 
 const defaultProps = {
     iou: iouDefaultProps,
+    selectedTab: '',
```

Comment:
@akinwale Do we even need an Empty string? We can just set the default value as null?

Reply:
@abdulrahuman5196 I assumed since we're using `PropTypes.oneOf` for the type, it had to be a literal value. I guess we can just change the type to `PropTypes.string`?

Reply:
>I assumed since we're using PropTypes.oneOf for the type, it had to be a literal value

I don't have a strong preference here, I did check our code base and we seem to have set `null` alone as default value with removing `isRequired` and also the way this PR is provided.
I think it would make a difference after TS migration but not ATM in this file I suppose. 

So I will review with the current change itself.